packages/plugin-serverless/src/utils.js

```diff
@@ -69,6 +69,17 @@ function normalizeFlags(flags, aliasMap) {
 function createExternalCliOptions(flags, twilioClient) {
   const profile = flags.profile;
 
+  if (flags.username || flags.password) {
+    return {
+      username: flags.username,
```

Comment:
Doesn't this mean `username` or `password` could potentially be null here? Is that okay?

Reply:
Realistically they wouldn't but I wrote a more proper check